contrib/labstack/echo.v4/echotrace.go

```diff
@@ -61,7 +65,7 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 			// serve the request to the next middleware
 			err := next(c)
 			if err != nil {
-				span.SetTag(ext.Error, err)
+				finishOptions = append(finishOptions, tracer.WithError(err))
```

Comment:
Why did you change this? Please undo this change. It is not necessary. It should be equal to the previous behaviour.

Reply:
The reason I added `finishOptions = append(finishOptions, tracer.WithError(err))` is due to this [line](https://github.com/DataDog/dd-trace-go/blob/v1.34.0/ddtrace/tracer/span.go#L308). If I don't add this, I don't believe that `cfg.NoDebugStack` (from finishOpts) will be taken into account.

This is why I decided to delete `span.SetTag(ext.Error, err)` as it seemed like it was duplicate behaviour to `finishOptions = append(finishOptions, tracer.WithError(err))` ?